apps/condo/domains/user/utils/testSchema/index.js

```diff
@@ -50,6 +50,8 @@ const { CHECK_USER_EXISTENCE_MUTATION } = require('@condo/domains/user/gql')
 const { ResetUserLimitAction: ResetUserLimitActionGQL } = require('@condo/domains/user/gql')
 /* AUTOGENERATE MARKER <IMPORT> */
 
+const OIDC_REDIRECT_URI = 'https://httpbin.org/anything'
```

Comment:
Is there any way to achieve same result with some local address?

Reply:
Or it's because oidc requires valid ssl cert?

Reply:
I will try it in separated task